include/mce/mode-names.h

```diff
@@ -239,5 +239,23 @@
  * @since v1.8.88
  */
 #define MCE_CABC_MODE_MOVING_IMAGE		"moving-image"
+/**
+ * POWERKEY EVENT for single powerkey press
+ *
+ * @since v1.10.54
+ */
+#define MCE_POWERKEY_SHORT_PRESS        (0u)
```

Comment:
Sorry about not noticing this earlier, but can you change the constants so that they are named MCE_POWERKEY_EVENT_xxx instead of MCE_POWERKEY_xxx.